src/bot.jl

```diff
@@ -0,0 +1,30 @@
+export precompileActivator, precompileDeactivator, precompilePather, @snoopiBot, @snoopiBenchBot, BotConfig
```

Comment:
Please respect the standard Julia naming conventions. See https://github.com/timholy/SnoopCompile.jl/pull/49#issuecomment-565013426.

Reply:
I don't know what I should do regarding this. What are your suggested names?

Reply:
Just some notes,

- variables should be like `precompile_activator` (same with file names)
- `fooBar` type of capitalization is never used
- `FooBar` should be used for structs (this seems to already be the case)